rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -18,6 +18,8 @@ include::modules/rosa-update-cli-tool.adoc[]
 [id="rosa-q3-2023_{context}"]
 === Q3 2023
 
+* **Documentation update:** The **Monitoring** section in the documentation has been expanded and now includes more detailed information to help you conveniently manage your ROSA clusters. For more information, see xref:../monitoring/monitoring-overview.html[Monitoring overview].
```

Comment:
```suggestion
* **Documentation update:** The Monitoring section in the documentation was expanded and now includes more details to help you conveniently manage your ROSA clusters. For more information, see xref:../monitoring/monitoring-overview.html[Monitoring overview].
```
Per [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=format-highlighting#highlighting-by-element), no highlighting is needed for the names of documentation sections. If it's standard to use bold to refer to documentation sections in the ROSA docs, feel free to disregard my suggestion. 

I replaced "has been" with "was" to use the simple present tense, per [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense).